src/Dfe.Spi.Common/Dfe.Spi.Common.Logging/Factories/LoggerWrapperFactory.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Dfe.Spi.Common.Logging.Factories
+{
+    using Dfe.Spi.Common.Logging.Definitions;
+    using Dfe.Spi.Common.Logging.Definitions.Factories;
+    using Dfe.Spi.Common.Models;
+    using Microsoft.Extensions.Logging;
+
+    /// <summary>
+    /// Implements <see cref="ILoggerWrapperFactory" />.
+    /// </summary>
+    public class LoggerWrapperFactory : ILoggerWrapperFactory
+    {
+        /// <inheritdoc />
+        public ILoggerWrapper Create(
+            ILogger logger,
```

Comment:
Can this not be in the constructor? Will avoid the need to inject both ILogger and ILoggerWrapperFactory in function